src/odin/directionsbuilder.cc

```diff
@@ -410,7 +370,7 @@ void DirectionsBuilder::PopulateDirectionsLeg(const Options& options,
     }
 
     // Travel mode
-    trip_maneuver->set_travel_mode(translate_travel_mode.find(maneuver.travel_mode())->second);
```

Comment:
everything else from here down is renaming stuff since i moved them out of directions and trip and into common